release.cmd

```diff
@@ -0,0 +1,32 @@
+@echo Off
+
+rem change working directory to the repository root
+pushd %~dp0
+setlocal
+
+rem options
+set NUGET_VERSION=4.1.0
+
+rem determine nuget cache dir
+set NUGET_CACHE_DIR=%LocalAppData%\.nuget\v%NUGET_VERSION%
+
+rem download nuget to cache dir
+set NUGET_URL=https://dist.nuget.org/win-x86-commandline/v%NUGET_VERSION%/NuGet.exe
+if not exist %NUGET_CACHE_DIR%\NuGet.exe (
+  if not exist %NUGET_CACHE_DIR% md %NUGET_CACHE_DIR%
+  echo Downloading '%NUGET_URL%'' to '%NUGET_CACHE_DIR%\NuGet.exe'...
+  @powershell -NoProfile -ExecutionPolicy unrestricted -Command "$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest '%NUGET_URL%' -OutFile '%NUGET_CACHE_DIR%\NuGet.exe'"
+)
+
+rem copy nuget locally
+if not exist .nuget md .nuget
+copy /Y %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
+
+rem restore packages for build script
+rem .nuget\NuGet.exe restore .\packages.config -PackagesDirectory .\packages -Verbosity quiet
+
+rem run build script
+".\packages\Microsoft.Net.Compilers.2.2.0\tools\csi.exe" %~n0.csx %*
+
+rem return to original working directory
+popd

```

Comment:
This script is nearly identical to build.cmd. (And probably needs a rename anyhow.)

We could consolidate and just pass in the name of the csx to run.
But then maybe we should talk about where to put "helper" scripts. I don't feel great about having them all litter the top level directory.

Reply:
> We could consolidate and just pass in the name of the csx to run.

I like the idea 👍 

> But then maybe we should talk about where to put "helper" scripts. I don't feel great about having them all litter the top level directory.

Yeah, it's already a bit of a mess... We could add a `build` or `scripts` directory.

Reply:
"build" always makes me wonder if that's where the builds go, but I'm not worried enough to fight either suggestion. If you've a favourite, that's fine.
Like the common `csx` we talk about below, do you want to wait, or would you rather have this now?

Reply:
Let's wait. We'll do it in a separate PR.